tests/queries/0_stateless/02884_string_distance_function.reference

```diff
@@ -2,6 +2,8 @@ const arguments byteHammingDistance
 0
 const arguments editDistance
 6
+const arguments jaccardIndex
```

Comment:
Unrelated: In docs/en/sql-reference/functions/string-search-functions.md, could you change `byteHammingDistance(string2, string2)` to `byteHammingDistance(string1, string2)`? I also think that the example (`abc, ab`) is not very illustrative.

It also seems weird in the docs that the Hamming distance (according to Wikipedia) requires the inputs to be equally long, but then the examples use different-length strings.